crates/stdarch-test/src/lib.rs

```diff
@@ -91,7 +91,7 @@ pub fn assert(_fnptr: usize, fnname: &str, expected: &str) {
 
     // Look for `expected` as the first part of any instruction in this
     // function, e.g., tzcntl in tzcntl %rax,%rax.
-    let found = instrs.iter().any(|s| s.contains(expected));
+    let found = instrs.iter().any(|s| s.starts_with(expected));
```

Comment:
No instruction in any `assert_instr` was changed, so I'm not sure why this is needed ? Is there an instruction that was matching exactly before but isn't matching exactly anymore ?

Reply:
Cause otherwise `rsqrt` matches when `sqrt` is expected.

Reply:
Makes sense, thanks for catching this!